tools/crm_mon.c

```diff
@@ -1720,7 +1721,9 @@ send_custom_trap(const char *node, const char *rsc, const char *task, int target
         out->err(out, "notification fork() failed: %s", strerror(errno));
     }
     if (pid == 0) {
-        /* crm_debug("notification: I am the child. Executing the nofitication program."); */
+        /* pcmk__debug("notification: I am the child. Executing the "
+         *             "notification program.");
+         */
```

Comment:
This message has been commented out since it was introduced in 2009 (938d01fe3c7).  I think we can get rid of it.